jest.config.js

```diff
@@ -54,7 +47,10 @@ module.exports = {
 		'\\.(scss)$': '<rootDir>/' + configPath + '/__mocks__/styleMock.js',
 		'\\.(jpg|jpeg|png|gif|eot|otf|webp|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
 			'<rootDir>/' + configPath + '/__mocks__/fileMock.js',
-		...transpiledPackages,
+		[ `@wordpress\\/(${ transpiledPackageNames.join(
+			'|'
+		) })$` ]: '<rootDir>/gutenberg/packages/$1/src',
```

Comment:
The logic for mapping the WordPress modules has been changed to do it the same way we do for the native unit tests:
- https://github.com/WordPress/gutenberg/blob/03d43d8424730212004cdf1ed842176ee9eec603/test/native/jest.config.js#L18-L20
- https://github.com/WordPress/gutenberg/blob/03d43d8424730212004cdf1ed842176ee9eec603/test/native/jest.config.js#L55-L57